src/types/viewport.cpp

```diff
@@ -922,8 +923,8 @@ Viewport Viewport::ToOrigin() const noexcept
         // |                          |             |                          |
         // |                          |             |                          |
         // |                          |             |                          |
-        // |                          |    ======>  |        intersect         |  ======>  early return of 0x0 Viewport
-        // |                          |             |                          |           at Original's origin
+        // |                          |    ======>  |        intersect         |  ======>  early return of nothing

```

Comment:
Technically we're not early returning anymore, right?

Reply:
Well there's a check for the empty case on line [913](https://github.com/microsoft/terminal/blob/7fbe60e1d2eadce8ba9e10aaf777049e00d997e6/src/types/viewport.cpp#L913), which skips to the end of the method where it returns nothing, thus avoiding the more complicated code of the general case. So while there isn't literally a return statement earlier in the method, it is technically returning earlier.